flow360/component/simulation/models/material.py

```diff
@@ -1,8 +1,17 @@
-from typing import Literal
+from typing import Literal, Optional, Union
 
 import pydantic as pd
+import unyt as u
```

Comment:
Can you do `import flow360.component.simulation.units as u`? Let's encapsulate the unyt package as much as possible.